OpenUtau.Core/Classic/VoicebankLoader.cs

```diff
@@ -317,7 +319,7 @@ public class VoicebankLoader {
                             otoSet.Otos.Add(oto);
                         }
                         if (!string.IsNullOrEmpty(oto.Error)) {
-                            Log.Error($"Failed to parse\n{trace}: {oto.Error}");
+                            Log.Error($"Failed to parse\n{oto.Error}");
```

Comment:
Why remove this? The whole point of `trace` is to help people see where the errors are from.

Reply:
That's because `trace` is included in `oto.Error`, so it's a double.